08-RAG-based-question-answering/01-question_answerIng_jumpstart_knn-apigateway.ipynb

```diff
@@ -632,19 +662,24 @@
    "outputs": [],
    "source": [
     "for model_id in _MODEL_CONFIG_:\n",
+    "    \n",
+    "    if num_context_doc == 0:\n",
+    "        print(f\"Sorry, I don't have information to answer this question:\\n{question}.\")\n",
+    "        break\n",
     "\n",
     "    prompt = _MODEL_CONFIG_[model_id][\"prompt\"]\n",
     "\n",
     "    text_input = prompt.replace(\"{context}\", context_retrieve)\n",
     "    text_input = text_input.replace(\"{question}\", question)\n",
-    "\n",
+    "    print(text_input, end=\"\\n\")\n",
```

Comment:
Suggest to put this print statement before the end. As what I observed for the second example is that there is a slight delay in generating the response from model, then for a few second I thought the model didn't generate anything. So suggest to put the two print together or in one print with \n to separate the paragraph

Reply:
Will update in the next round